application/src/main/webapp/vue-app/components/ExoChatMessageDetail.vue

```diff
@@ -295,6 +306,14 @@ export default {
     messageType() {
       return this.message && this.message.type;
     },
+    isExternalLink() {
+      if (this.isURL(this.messageContent)) {
+        const domain = (new URL(this.messageContent));
+        return !(domain.hostname === window.location.hostname);
```

Comment:
Use URL.host instead of URL.hostname to consider the port value https://developer.mozilla.org/en-US/docs/Web/API/URL/host
URL = http://myserver:8080
URL.hostname = myserver
URL.host = myserver:8080
```suggestion
        return !(domain.host === window.location.host);
```

Reply:
Okay